zingolib/src/commands.rs

```diff
@@ -1031,32 +1031,6 @@ impl Command for SeedCommand {
     }
 }
 
-struct TransactionsCommand {}
```

Comment:
Could this also be deprecated, instead of being fully removed? While I can use this in the integration tests (assuming that's still the plan here?) without the command interface, being able to do it in a manual test as well would be useful.

Reply:
OK, I will put it back in.